BUILDING.md

```diff
@@ -668,7 +668,7 @@ packages:
 
 * [Git for Windows](https://chocolatey.org/packages/git) with the `git` and
   Unix tools added to the `PATH`
-* [Python 3.x](https://chocolatey.org/packages/python)
+* [A supported version of Python][Python versions]
```

Comment:
Let's revert because the supported Python versions are stated above and the current comment recommends where to install Python from.
```suggestion
* [Python 3.x](https://chocolatey.org/packages/python)
```